Source/UI/CommonUI/View/Core/ButtonView.swift

```diff
@@ -31,6 +37,8 @@ class ButtonView: UIView {
         LabelView().disabledAutoresizingIntoConstraints()
     }()
 
+    private var loadingOverlay: UIView?
```

Comment:
I see there is a edge case in your videos where user tries to update card details when the pay button is pressed ? Are we not disabling the screen when pay button is pressed ?. Just want to understand what is the expected behaviour in this case.

Reply:
No, we are not disabling the screen while pending the tokenisation request and we are not communicating anything to application owner either, so they could not block UI on their side.

There was an activity indicator placed in the navigation controller but it would not prevent multiple requests being triggered.

[Recent Slack discussion](https://checkout.slack.com/archives/GS5KC6HU3/p1664870400500179)

Reply:
@deepesh-vasthimal-cko I will merge this for now as it prevents unexpected behaviours from occurring, but happy to change again once @patrick-hoban-cko is back if you guys want a different solution.

I'd just like to limit possibilities that things go wrong for the user for now, and I think you agreed, it does achieve that